bin/improver-nbhood-iterate-with-mask

```diff
@@ -45,10 +45,14 @@ def main():
     parser = ArgParser(
         description='Apply the requested neighbourhood method via '
                     'the ApplyNeighbourhoodProcessingWithAMask plugin '
-                    'to a file with one cube using a mask. The masking '
-                    'coordinate is iterated over, so that different masks '
-                    'as defined by the masking coordinate can be applied to '
-                    'the cube that is being neighbourhood processed.')
+                    'to a file with one cube using a mask. Within the mask '
+                    'cube, the coordinate chosen as the coordinate to mask '
+                    'with, is iterated over. Therefore every slice of the '
+                    'coordinate to mask with is applied to the input cube '
+                    'as it is neighbourhood processed. There is also '
+                    'an option to re-mask the output cube, so that after '
+                    'neighbourhood processing, non-zero values are only '
+                    'present for grid points that are masked in. ')
```

Comment:
`Masked in` is a rather ambiguous phrase (if it is a phrase?). 'only present for unmasked points'?

Reply:
Updated phrasing.